typescript/looker/sdk/4.0/models.ts

```diff
@@ -22,6 +22,8 @@ export interface IAccessToken {
    * Number of seconds before the token expires (read-only)
    */
   expires_in?: number
+
+  refresh_token?: string
```

Comment:
tinkering with a generated file here. Need to make a corresponding PR to Looker to survive regen.